app/src/main/java/com/android/voyageur/model/trip/TripsViewModel.kt

```diff
@@ -550,4 +550,14 @@ open class TripsViewModel(
     tripInviteRepository.createTripInvite(
         req = tripInvite, onSuccess = onSuccess, onFailure = onFailure)
   }
+
```

Comment:
Could you please add documentation for this function? Saves us some time for the M3 deadline.

Reply:
Thanks for pointing this out! Done!